apps/console/src/public/deployment.config.json

```diff
@@ -305,7 +305,10 @@
             },
             "actions": {
                 "disabledFeatures": [
-                    "actions.types.list.preRegistration"
+                    "actions.types.list.preRegistration",
+                    "actions.types.org.list.preIssueAccessToken",
+                    "actions.types.org.list.preRegistration",
+                    "actions.types.org.list.preUpdateProfile"
```

Comment:
Let's update framework too if we want to keep this disabled in the pack

Reply:
This was updated with https://github.com/wso2/carbon-identity-framework/pull/7031